website/Nav.tsx

```diff
@@ -44,62 +51,73 @@ const rtlCheckboxClassname = css`
 `;
 
 interface Props {
+  headerId: string;
   direction: Direction;
   onDirectionChange: (direction: Direction) => void;
 }
 
-export default function Nav({ direction, onDirectionChange }: Props) {
-  return (
-    <nav className={navClassname}>
-      <h1>react-data-grid</h1>
+export default function Nav({ headerId, direction, onDirectionChange }: Props) {
+  const demosNavId = useId();
+  const linksNavId = useId();
 
-      <h2>Demos</h2>
-      <Link to="/CommonFeatures">Common Features</Link>
-      <Link to="/AllFeatures">All Features</Link>
-      <Link to="/CellNavigation">Cell Navigation</Link>
-      <Link to="/ColumnSpanning">Column Spanning</Link>
-      <Link to="/ColumnGrouping">Column Grouping</Link>
-      <Link to="/ColumnsReordering">Columns Reordering</Link>
-      <Link to="/ContextMenu">Context Menu</Link>
-      <Link to="/CustomizableRenderers">Customizable Renderers</Link>
-      <Link to="/RowGrouping">Row Grouping</Link>
-      <Link to="/HeaderFilters">Header Filters</Link>
-      <Link to="/InfiniteScrolling">Infinite Scrolling</Link>
-      <Link to="/MasterDetail">Master Detail</Link>
-      <Link to="/MillionCells">A Million Cells</Link>
-      <Link to="/NoRows">No Rows</Link>
-      <Link to="/ResizableGrid">Resizable Grid</Link>
-      <Link to="/RowsReordering">Rows Reordering</Link>
-      <Link to="/ScrollToCell">Scroll To Cell</Link>
-      <Link to="/TreeView">Tree View</Link>
-      <Link to="/VariableRowHeight">Variable Row Height</Link>
-      <Link to="/Animation">Animation</Link>
+  return (
+    <header className={headerClassname}>
+      <h1 id={headerId}>react-data-grid</h1>
+      <nav aria-labelledby={demosNavId} className={navClassname}>
```

Comment:
Added accessible name so screen reader can announce it like `Demos nav landmark`